clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleHttpLoadBalancerCachingAgent.groovy

```diff
@@ -211,13 +214,54 @@ class GoogleHttpLoadBalancerCachingAgent extends AbstractGoogleCachingAgent impl
               httpHealthChecksRequest: httpHealthChecksRequest,
               groupHealthRequest: groupHealthRequest,
           )
+          def targetHttpsProxyCallback = new TargetHttpsProxyCallback(
+              googleLoadBalancer: newLoadBalancer,
+              urlMapRequest: urlMapRequest,
+              backendServiceRequest: backendServiceRequest,
+              httpHealthChecksRequest: httpHealthChecksRequest,
+              groupHealthRequest: groupHealthRequest,
+          )
 
+          // We make two requests here with the same target proxy name -- this will cause problems if there is
+          // a Http and Https target proxy with the same name. This shouldn't happen in practice, since Spinnaker
+          // will control the load balancer namespace.
```

Comment:
I check int the `UrlMapCallback` to see if the LB has any data populated yet, and log an error message if it does. Should address this pretty well.